### PR TITLE
Task18.1

### PR DESCRIPTION
**Pull request by uthag1**
check please

### CODE REVIEW

File: src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/DashboardView.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
 import com.vaadin.flow.server.PWA;
 
 
-@Route("")
```
Comment (adams3-sys) [inline src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/DashboardView.java]: Route should point to the DashboardView, it is empty here

File: src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/DashboardView.java [hunk 2]
```diff
@@ -15,7 +15,7 @@
...
+@Route("ghgh")
```
Comment (adams3-sys) [inline src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/DashboardView.java]: what is this bro?

File: src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/InfoboxView.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+		title1 = new H4();
+		title1.setText("Wichtige Personen");
+		super.addContent(title1);
+
+		Button btnImportantPer = new Button("Add User");
+		super.addContent(btnImportantPer);
+
+		Accordion accordion1 = new Accordion();
+
+		VerticalLayout importentNr = new VerticalLayout();
```
Comment (adams3-sys) [inline src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/InfoboxView.java]: why the fuck you mix german and english you little weasel?

File: src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/InfoboxView.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+import com.vaadin.flow.router.Route;
+
+/**
+ * The user management view will be used to add and remove users of the app
+ */
+@Route("")
+@CssImport(value = "./styles/shared-styles.css", include = "common-styles")
+public class InfoboxView extends TemplateView {
+
+	private H4 title1;
```
Comment (adams3-sys) [inline src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/InfoboxView.java]: you could use a bit more informative name declarations

**Comment by uthag1** [comment]
thank you

File: src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/DashboardView.java (hunk 1 above)
Comment (uthag1) [reply src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/DashboardView.java]: gratuliere :)

File: src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/DashboardView.java (hunk 2 above)
Comment (uthag1) [reply src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/DashboardView.java]: gratuliere :)

File: src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/InfoboxView.java (hunk 3 above)
Comment (uthag1) [reply src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/InfoboxView.java]: gratuliere :)

File: src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/InfoboxView.java (hunk 4 above)
Comment (uthag1) [reply src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/InfoboxView.java]: gratuliere :)

**Pull request merged**